space-cli/cmd/modules/accounts/set_test.go

```diff
@@ -0,0 +1,386 @@
+package accounts
+
+import (
+	"errors"
+	"testing"
+
+	"github.com/AlecAivazis/survey/v2"
+	"github.com/spaceuptech/space-cli/cmd/utils"
+	"github.com/spaceuptech/space-cli/cmd/utils/file"
+	"github.com/spaceuptech/space-cli/cmd/utils/input"
+)
+
+func Test_setAccount(t *testing.T) {
+	// singleMatchCase stores the value returned from survey when prefix mathes one account ID
```

Comment:
```suggestion
	// singleMatchCase stores the value returned from the survey when prefix matches one account ID
```
change mathes to matches